src/pages/module-builder/Toolbar.tsx

```diff
@@ -62,6 +67,17 @@ const Toolbar = ({
             </Button>
             <Spacer />
             <HStack spacing={2}>
+                <IconButton
+                    variant="ghost"
+                    aria-label="undo"
```

Comment:
misleading aria label? should be something like "previous slide" or just remove it entirely